### ISSUE TITLE
Deep Learning Hands on workshop

### ISSUE THREAD

**Issue by sumodm** [sumodm/bcvm#2]

**Comment by utkarshsimha** [comment]
Maybe we can start of with [Softmax Regression on TensorFlow](https://www.tensorflow.org/versions/r0.7/tutorials/mnist/beginners/index.html) to start off with? It is very simple and can be completed quickly also. It gives a solid introduction to the basic of classification and softmax function.
But instead of using the cliched MNIST dataset, we can try and look into the alphabets one maybe?

**Comment by sumodm** [comment]
+1 for that :)
